src/Home.jsx

```diff
@@ -2,17 +2,49 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
+import Select from 'react-select';
 import Layout from './Layout';
+import RestAPI from './commons/RestAPI';
 
 class Home extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      activeBlock: null,
-      toggledBar: true,
+      data: [],
+      selectedElement: null,
+      selectedArea: null,
+      toggledBar: null,
+      sedimentaryList: [],
     };
   }
 
+  async componentDidMount() {
+    try {
+      const areasResponse = await RestAPI.requestANHAreas();
+      const sedimentaries = await RestAPI.requestSedimentaryBasins();
+      const arrayAreas = areasResponse.map((item) => item.name);
+      const arraySedimentaries = sedimentaries.map((item) => item);
+      this.setState({
+        toggledBar: false,
+        data: arrayAreas.map((element) => ({
+          name: element,
+          label: element,
+        })),
+        sedimentaryList: arraySedimentaries,
```

Comment:
Same here, since you are nor doing anything with `arraySedimentaries`, you could map directly `sedimentaries`, but `arraySedimentaries` is exactly `sedimentaries`, so you actually don't need to map anything

Reply:
Thank you. Done.